content/projects/gsoc/2023/project-ideas/alternative-jenkinsio-build-tool.adoc

```diff
@@ -36,6 +36,12 @@ The preferred tool to replace Awestruct is link:https://antora.org/[Antora].
 The potential GSoC project would be to build a working site generator to demonstrate the existing site.
 Once the existing site is generated with Antora, the site should be extended to add version specific documentation.
 
+The project has been discussed extensively at link:https://github.com/jenkins-infra/jenkins.io/issues/5474[GitHub issue #5474], where some existing proof-of-concept code can be found referenced there.
+
+There are multiple ways to approach the implementation, but from experimentation it has been found that the backend replacement requires minimal effort for the documentation, with the frontend implementation expected to require much effort to reproduce the look and feel of the current link:https://www.jenkins.io/[jenkins.io] website. However, the blog can be split from the documentation using something like link:https://www.gatsbyjs.com/[Gatsby], which is expected to make it easier for users to submit posts in the future.
```

Comment:
The scope of this project is still ambiguous to me after reading this project description. Open questions include whether this project covers the blog (Gatsby), the user documentation (Antora), and the developer documentation (Antor) or a subset of these three, as well as what the final deliverable should be (a drop-in replacement web site building locally? a drop-in replacement web site deployed to production?).

Reply:
Let me add a paragraph regarding the scope soon.

Reply:
Done